src/components/utils/custom-dialog.tsx

```diff
@@ -36,11 +39,13 @@ export const CustomDialog: FunctionComponent<CustomDialogProps> = ({
             {title && <DialogTitle id={'modal-title'}>{title}</DialogTitle>}
             <DialogContent>{content}</DialogContent>
             <DialogActions>
-                <Button onClick={handleClose}>
-                    {intl.formatMessage({ id: 'cancel' })}
-                </Button>
+                <CancelButton onClick={handleClose} />
                 <Button variant={'outlined'} onClick={handleValidate}>
-                    {intl.formatMessage({ id: 'validate' })}
+                    {intl.formatMessage(
```

Comment:
sometimes we have` <FormattedMessage>` component sometimes just a string with `intl.formatMessage,` we should avoid the later I think, I know it's not your code but I saw it !

Reply:
Yes, to do in another ticket.